docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -828,13 +828,13 @@ Note that there is no dependency on Mockito, you can use any mocking library you
 objects to provide the behaviour you require.
 
 NOTE: Using `@Inject` will get you a CDI proxy to the mock instance you install, which is not suitable for passing to methods such as `Mockito.verify`
-which want the mock instance itself. So if you need to call methods such as `verify` you need to hang on to the mock instance in your test, or use `@InjectMock`
+which want the mock instance itself. So if you need to call methods such as `verify` you need to hang on to the mock instance in your test, or use `@io.quarkus.test.InjectMock`
```

Comment:
I suggest 

"So if you call methods such as `verify` you should hang on to the mock instance in your test, or use `@io.quarkus.test.InjectMock`"